src/gui/optionsdialog.ui

```diff
@@ -232,7 +235,7 @@
                <item>
                 <widget class="QCheckBox" name="checkAltRowColors">
                  <property name="text">
-                  <string extracomment="In transfer list, one every two rows will have grey background.">Use alternating row colors</string>
+                  <string extracomment="In table elements, one every two rows will have a grey background.">Use alternating row colors</string>
```

Comment:
"one every two rows ..." is that valid grammar? Could some one verify?

Reply:
I believe "one every two" is grammatically correct, despite how improper it sounds. 

Maybe "In table elements, every consecutive row will have an alternating color?" I'm not sure if the purpose of the extracomment here is to help understand the word 'alternating' or the phrase as a whole.

Reply:
@NotTsunami, @Chocobo1: 

> I believe "one every two" is grammatically correct, despite how improper it sounds. 

Technically, the correct phrasing in English is **"every other row"**.

Reply:
@NotTsunami @Chocobo1 
Then would you prefer:
`In table elements, every consecutive row will have an alternating color.`
or
`In table elements, every other row will have a grey background.`

Reply:
The latter one seems better to me.

Reply:
@Chocobo1, @NotTsunami: 

> The latter one seems better to me. 

Likewise.

Reply:
Thanks, went with the latter.